pkg/config/config_darwin.go

```diff
@@ -19,8 +19,10 @@ func ifRootlessConfigPath() (string, error) {
 var defaultHelperBinariesDir = []string{
 	// Homebrew install paths
 	"/usr/local/opt/podman/libexec",
+	"/usr/local/opt/podman/libexec/bin",
 	"/opt/homebrew/bin",
 	"/opt/homebrew/opt/podman/libexec",
+	"/opt/homebrew/opt/podman/libexec/bin",
 	"/usr/local/bin",
```

Comment:
> can you please elaborate on this? this is something maybe we didn't know about?

Take https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/qemu.rb#L72 for example, Homebrew tells qemu where to find Samba, saving the developers of QEMU from maintaining an ever outdated list of all the places someone could install Samba. Almost all packaged software in the world uses this system with the exception of podman it seems, because here we are updating an ever outdated list of hardcoded paths.

Reply:
We do not use ./configure scripts with golang, so you cannot set a prefix like that.
https://github.com/containers/podman/pull/13372 added option to set a full path at compile time, I think this should be good enough?

Reply:
> We do not use ./configure scripts with golang, so you cannot set a prefix like that.

There's ways to do this with Golang too AFAIK, the configure was just an example.

> https://github.com/containers/podman/pull/13372 added option to set a full path at compile time, I think this should be good enough?

If that affects the same settings then that would roughly be the solution I proposed 👍